### ISSUE TITLE
Add the option to install dump978.

### ISSUE THREAD

**Issue by jprochazka** [jprochazka/adsb-feeder#27]
This is something I want to add to my existing feeder once I get my 978MHz antenna in the air. The final goal will be to add the ability to install dump978 on the same device as dump1090 using a second RTL-SDR dongle to process 978MHz UAT data. Look for this to be added once I figure out the ins and outs of running a second dongle on the same device running dump1090.

**Comment by jprochazka** [comment]
I am looking for a way to listr connected devices.
Running rtl_test outputs the following:

    Found 2 device(s):
      0:  Generic, RTL2832U, SN: 77771111153705700
      1:  Realtek, RTL2838UHIDIR, SN: 00000001

    Using device 0: Generic RTL2832U
    usb_claim_interface error -6
    Failed to open rtlsdr device #0.

The problem is I just want to top portion I do not want to actually test, flash, listen to, etc..
I just need the list of devices.

Anyone know a way to accomplish this?

**Comment by dschaper** [comment]
Just adding another data point for you, for testing.

```
orangepi@OrangePI:~/dump978$ rtl_test
Found 2 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001
  1:  Realtek, RTL2838UHIDIR, SN: 00000001

Using device 0: Generic RTL2832U OEM
usb_claim_interface error -6
Failed to open rtlsdr device #0.
```

**Comment by jprochazka** [comment]
Right there is the information I need.
      0:  Realtek, RTL2838UHIDIR, SN: 00000001
      1:  Realtek, RTL2838UHIDIR, SN: 00000001
Only problem is it is trying to claim the device to use it which I do not want...
    Using device 0: Generic RTL2832U OEM
    usb_claim_interface error -6
    Failed to open rtlsdr device #0.

**Comment by dschaper** [comment]
Yeah, I had to hard code in device 0 for 1090 and device 1 for 978.

**Comment by dschaper** [comment]
Oh, I misunderstood what you are trying to do... I see you just want to list the devices. sorry.

**Comment by jprochazka** [comment]
Right that's why I am trying to get a list of devices so a person can choose which device receives 1090MHz and which receives 978MHz data. Everything is in place to accomplish this in the script I just need a way to display the devices and their index to the user so they can choose which goes where.

**Comment by dschaper** [comment]
A call to `dump1090-mutability` or `dump978` should list the devices, would that be of any use?

**Comment by jprochazka** [comment]
Looking for anyone willing to test out the dump978 branch. Installation appears to be good however I do not receive enough UAT data to do any real world testing at the moment. Once I receive feedback or actually receive any data on my currently indoor 978MHz antenna I can get any needed fixes in place and/or merge this branch into the master branch.

On clean installations:

    sudo apt-get install git
    cd ~/
    git clone https://github.com/jprochazka/adsb-feeder.git
    cd ~/adsb-feeder
    git checkout dump978
    chmod +x install.sh
    ./install.sh

On existing installations:

    cd ~/adsb-feeder
    git fetch --all
    git reset --hard origin/master
    git checkout dump978
    ./install.sh

**NOTE:**
I am still unable to find a way to list the rtl-sdr devices along with their indexes yet. However when asked assigning 0 to dump1090 will assign the first device. Assigning 1 to dump978 will assign the second device to dump978.

I will be updating the wiki to explain these instruction better at least until a way to list the devices is found.

**Comment by dschaper** [comment]
Okay, I'm testing it now. Did get the following error:
```
Restarting dump1090-mutability (via systemctl): dump1090-mutability.serviceWarning: dump1090-mutability.service changed on disk. Run 'systemctl daemon-reload' to reload units.
```

And when the script ends it goes to dump978 screen, not a command prompt. (I get about 4 or 5 tracks a day here with my indoor antenna...)

**Comment by jprochazka** [comment]
I have got that error in the past when doing restarts even those unrelated to the scripts. Run the command as it says and the error will go away.

    sudo systemctl daemon-reload

I see where the script is dumping out it. It happens when only dump978 is chosen to be installed. I just committed a fix which is not tested. In the process of rolling back my test VM.

**Comment by dschaper** [comment]
Yeah, I hit enter and got to the final screen. I rebooted anyways because I uncommented the logging feature in the maintenance script just so I could track the /tmp file. I have it open in a byobu screen window to make sure it's seeing tracks.

**Comment by dschaper** [comment]
I have gpsd running so I'm testing Piaware 2.2.0~dev at the same time, things look okay so far.

**Comment by jprochazka** [comment]
I should have thought to uncomment the logging lines for testing... Anyways I just uncomment them. Dump978 output "should" be logged to the file /tmp/dump978.out from now on until it is deemed ready to be merged.

**Comment by dschaper** [comment]
Looks good, tracked a Cessna as it came into KSNA, showed on the dump1090 map with no problems, and the log in /tmp shows the downlink traffic.

**Comment by jprochazka** [comment]
Great to hear and thanks once again for your help. Will be merging it shortly after I get the wiki documentation done.

**Comment by jprochazka** [comment]
Merged with master #59
Thanks to @dschaper for all the help.
